public/bundle.js

```diff
@@ -68,7 +68,7 @@
 
```

Comment:
add your bundle file to .gitignore. you don't want them showing up in pull requests